dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/ValidationErrorReporter.java

```diff
@@ -89,7 +90,11 @@ public ValidationErrorReporter( TrackerIdSchemeParams idSchemes, boolean failFas
     {
         this.errors = new ArrayList<>();
         this.warnings = new ArrayList<>();
-        this.invalidDTOs = new EnumMap<>( TrackerType.class );
+        this.invalidDTOs = new EnumMap<>( Map.of(
```

Comment:
Since it's an enum map of a collection this initializes the collections. This way you can safely check containment or add to the collection of a type.